discordbot.js

```diff
@@ -199,19 +206,66 @@ exports.createDiscordClient = (web3, contracts, getStores, runSidechainTransacti
     if (discordApiToken === undefined || discordApiToken === "" || discordApiToken === null)
         return console.warn("*** WARNING: Discord API token is not defined");
 
-    const client = new Discord.Client();
-
+    const client = new Discord.Client({ partials: ['MESSAGE'] });
```

Comment:
Does this mean many other commands must now handle partial message fetch?

Reply:
Removed

Reply:
If it's not needed, why was it there?

Reply:
The examples I was looking at had it, but seems to work without